docker/docker-compose-quickstart/docker-compose-postgres.yml

```diff
@@ -53,10 +53,159 @@ services:
     volumes:
       - es-data:/usr/share/elasticsearch/data
 
+  execute-migrate-all:
+    container_name: execute_migrate_all
+    image: docker.getcollate.io/openmetadata/server:1.1.0
+    command: "./bootstrap/bootstrap_storage.sh migrate-all"
+    environment:
```

Comment:
@dhruvinmaniar123 Add an extra variable here - 

```
MIGRATION_LIMIT_PARAM: ${MIGRATION_LIMIT_PARAM:-1200}
```

This is the environment variable which will govern how many records to consider for migrations!
cc - @harshach @mohityadav766